### PR TITLE
Support Hearthbuddy 0.3.1267.298

### PR DESCRIPTION
**Pull request by x014**
Support Hearthbuddy 0.3.1267.298

### CODE REVIEW

**Comment by sliverfishes** [comment]
Hmm, I guess I can't use c# 6 sugar with HB. I have some nitpicking with the PR though and because I'm in a bit of a rush with kara I will give you a few hours to make the changes if you want to before I do it myself.

Please remove the 7zip and version # change, I will package it myself.

File: ai/Playfield.cs [hunk 1]
```diff
@@ -4353,7 +4359,7 @@ public void triggerACardWillBePlayed(Handmanager.Handcard hc, bool own, Minion t
 
                 for (int i = 0; i < summonstones; i++)
                 {
-                    summoningStone?.handcard.card.sim_card.onCardIsGoingToBePlayed(this, hc.card, own, summoningStone, target, choice);
+                    summoningStone.handcard.card.sim_card.onCardIsGoingToBePlayed(this, hc.card, own, summoningStone, target, choice);
```
Comment (sliverfishes) [inline ai/Playfield.cs]: this still needs a null check

File: ai/Playfield.cs [hunk 2]
```diff
@@ -4441,7 +4447,7 @@ public void triggerACardWillBePlayed(Handmanager.Handcard hc, bool own, Minion t
...
```
Comment (sliverfishes) [inline ai/Playfield.cs]: and null check here too

**Comment by x014** [comment]
Revert the 7zip and version # change yet

**Comment by sliverfishes** [comment]
What happened to all the null check changes? Is c# 6 null propagation with ?. okay or not?

**Comment by sliverfishes** [comment]
Also whenever it is ready please create a new branch, cherry-pick the commits that matter from your master to the new branch, and then PR the new branch.

**Comment by x014** [comment]
in Hearthbuddy,  c# 6 can't be identified. so  report errors.

**Comment by x014** [comment]
sorry, please close this pull request.  i create a new branch, then pull new request.

**Pull request closed without merging**